android/src/main/java/com/bugsnag/BugsnagReactNative.java

```diff
@@ -46,14 +46,23 @@ public static ReactPackage getPackage() {
   }
 
   public static Client start(Context context) {
-    return Bugsnag.init(context);
+    Client client = Bugsnag.init(context);
+    // The first session starts during JS initialization
+    // Applications which have specific components in RN instead of the primary
+    // way to interact with the application should instead leverage startSession
+    // manually.
+    client.setAutoCaptureSessions(false);
+    return client;
   }
 
   public static Client startWithApiKey(Context context, String APIKey) {
-    return Bugsnag.init(context, APIKey);
+    Client client = Bugsnag.init(context, APIKey);
+    client.setAutoCaptureSessions(false);
```

Comment:
It would be good to consolidate calls to `client.setAutoCaptureSessions(false)` so that they only occur in one method.

Reply:
I didn't notice the difference between config/client at first - this should be fine as-is, as long as it remains only 1 line of configuration.